src/Utilities/Compiler/Extensions/IOperationExtensions.cs

```diff
@@ -741,7 +741,7 @@ public static IOperation WalkDownConversion(this IOperation operation, Func<ICon
             return operation;
         }
 
-        public static ITypeSymbol? GetThrownExceptionType(this IThrowOperation operation)
+        public static IOperation? GetThrownException(this IThrowOperation operation)
```

Comment:
I see the code here is existing prior to the PR but wanted to make sure it is correct.

Should this method handle an explicit conversion? e.g, what happens for `throw (Exception)new ArgumentNullException())`

Could it just be `WalkDownConversion()` (which `UseExceptionThrowHelpers` was already using).

Reply:
Good point, let me update the code.